config/samples/dataplane_v1beta1_openstackdataplane_baremetal_with_ipam.yaml

```diff
@@ -0,0 +1,165 @@
+apiVersion: dataplane.openstack.org/v1beta1
```

Comment:
This symlinked file has already been dropped  with https://github.com/openstack-k8s-operators/dataplane-operator/pull/401/files

Reply:
Fixed this one too.